config/locales/ar/organisations.yml

```diff
@@ -90,6 +90,7 @@ ar:
     services_and_information: جميع معلومات وخدمات %{acronym}
     subscription:
       email: احصل على رسائل البريد الإلكتروني
+      email_english_only: احصل على رسائل البريد الإلكتروني
```

Comment:
I can't find a link to guidance, but I don't think we use generated translations. If we don't have a translation we add the key with an empty string. Please could you amend?

Reply:
Fixed